python/interpret-core/interpret/glassbox/_ebm/_ebm.py

```diff
@@ -2467,107 +2224,16 @@ class ExplainableBoostingClassifier(EBMModel, ClassifierMixin, ExplainerMixin):
         Bagged intercept of the model. Binary classification is shape ``(n_outer_bags,)``, and multiclass is shape ``(n_outer_bags, n_classes)``.
     """
 
-    n_features_in_: int
-    term_names_: List[str]
-    bins_: List[Union[List[Dict[str, int]], List[np.ndarray]]]  # np.float64, 1D[cut]
-    feature_names_in_: List[str]
-    feature_types_in_: List[str]
-    feature_bounds_: np.ndarray  # np.float64, 2D[feature, min_max]
-    term_features_: List[Tuple[int, ...]]
-    bin_weights_: List[np.ndarray]  # np.float64, [bin0...]
-    bagged_scores_: List[np.ndarray]  # np.float64, [bag, bin0..., ?class]
-    term_scores_: List[np.ndarray]  # np.float64, [bin0..., ?class]
-    standard_deviations_: List[np.ndarray]  # np.float64, [bin0..., ?class]
-    link_: str
```

Comment:
link_ could be a Litteral, although we might add new possible Litteral values in successive versions.

Reply:
Here's the list of possible link values.  Most of these are currently not used, but they could be someday:

https://github.com/interpretml/interpret/blob/d5ec3fa04db8e2acc5fc0de6ca00ff396852caec/shared/libebm/DetermineLinkFunction.cpp#L138-L156

Reply:
and here is a list of the link functions we currently implement:

https://github.com/interpretml/interpret/blob/d5ec3fa04db8e2acc5fc0de6ca00ff396852caec/python/interpret-core/interpret/utils/_link.py#L11-L16